### ISSUE TITLE
"Unfinished Fonts"

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1573]
Chatting with @eliheuer and @xuv about @n8willis ideas about what GF hasn't traditionally done, and the lgm 2014 (?) Madrid presentation by @rlafuente and @aiscarvalho, about how many designers have unfinished typefaces on their harddisks that go unpublished because it's a huge amount of work to finish them. 

I wonder if we could collect here a new milestone group for such "Unfinished Fonts" and solicit such new publications of them under the OFL

**Comment by felipesanches** [comment]
I love this idea!

**Comment by eliheuer** [comment]
Yeah, this is a great idea, here are mine, I also have an Optima style variable font drawn in quadratics on my HD that isn't in git yet. 

https://github.com/eliheuer/epistle
https://github.com/eliheuer/tenpo
https://github.com/eliheuer/woodbine
https://github.com/eliheuer/behrens-antiqua
https://github.com/eliheuer/isotherma
https://github.com/eliheuer/ren
https://github.com/eliheuer/tokivar
https://github.com/eliheuer/HeuerSchrift
https://github.com/eliheuer/loje
https://github.com/eliheuer/upm256

**Comment by laerm0** [comment]
Wow @eliheuer , that's a good stack to start on. I have a handful as well but I'm too possessive to let someone else at them (yet). 😂

**Comment by davelab6** [comment]
https://github.com/google/fonts/milestone/15
